resources/carrier/en/32.txt

```diff
@@ -15,11 +15,10 @@
 # 1994 by Belgacom, following the launch of the first mobile network.Proximus is
 # a subsidiary of Belgacom.
 # Mobistar has re-branded as Orange on 9 May 2016.
-# Unleashed has 3 brands Mobile Vikings, JIM Mobile and Stievie. Prefix 456
-# phone numbers are mentioned in contact-us page of JIM mobile carrier. So it is
-# more likely that it is assigned to 'JIM Mobile'
+# Unleashed has 2 mobile brands: Mobile Vikings and JIM Mobile. Both of these
```

Comment:
Unleashed website https://unleashed.be/ still says that it has 3 brands 'Mobile Vikings, JIM Mobile and Stievie.'

Reply:
Also as stated in the commit message, Stievie is not a mobile carrier (it's a tv app) and has nothing to do with phone numbers at all. Note that I changed the wording from "3 brands" to "2 mobile brands".

Reply:
This is perfectly reasonable.